valueset.json

```diff
@@ -19,7 +19,37 @@
     },
     "valueSetValues": {
       "title": "Allowed values in Value Set",
-      "type": "object"
+      "type": "object",
+      "patternProperties": {
+        "^.*$": {
+          "title": "Allowed values in records",
+          "type": "object",
+          "properties": {
+            "display": {
+              "type": "string"
+            },
+            "lang": {
+              "type": "string"
+            },
+            "active": {
+              "type": "boolean"
+            },
+            "version": {
+              "type": "string"
+            },
+            "system": {
+              "type": "string"
+            }
+          },
+          "required": [
+            "display",
+            "lang",
+            "active",
+            "version",
```

Comment:
as per https://github.com/ehn-dcc-development/ehn-dcc-valuesets/issues/22#issuecomment-951661634

we should consider the case where the version might be an empty string

Reply:
We definitely should cover that in a test case. I'm pretty sure that an empty string satisfies the schema w.r.t. “'version' must be a string”, though.

Reply:
Totally agree with the test case.